atom/browser/api/event_emitter.cc

```diff
@@ -77,12 +77,15 @@ v8::Local<v8::Object> CreateCustomEvent(v8::Isolate* isolate,
 }
 
 v8::Local<v8::Object> CreateEventFromFlags(v8::Isolate* isolate, int flags) {
+  int click_flags = (ui::EF_RIGHT_MOUSE_BUTTON | ui::EF_LEFT_MOUSE_BUTTON |
+                     ui::EF_MIDDLE_MOUSE_BUTTON);
+  int mask = flags & click_flags;
```

Comment:
_(nit)_ It's a little confusing that the mask is named 'flags', the remaining flags are named 'mask', and that mask is not very descriptive when used below. I think we could simplify this:

```cc
const int mouseButtonMask = ui::EF_RIGHT_MOUSE_BUTTON | ...;
const bool isMouseClick = static_cast<bool>(flags & mouseButtonMask);
...
obj.set("triggeredByAccelerator", !isMouseClick);
```

To be clear, I think the code will work as-is. This suggestion is about making the code as straightforward as possible.